src/maud/cli.py

```diff
@@ -87,6 +87,58 @@ def sample_command(data_path, output_dir):
     click.echo(sample(data_path, output_dir))
 
 
+def ppc(samples_path, ppc_path, output_dir):
+    """Generate MCMC samples given a user input directory.
+
+    This function creates a new directory in output_dir with a name starting
+    with "maud_output". It first copies the directory at data_path into the new
+    this directory at new_dir/user_input, then runs the sampling.sample
+    function to write samples in new_dir/samples. Finally it prints the results
+    of cmdstanpy's diagnose and summary methods.
+
+    """
+    csvs = [
+        os.path.join(samples_path, "samples", f)
+        for f in os.listdir(os.path.join(samples_path, "samples"))
+        if f.endswith(".csv")
+    ]
+    mi_oos = load_maud_input_from_toml(ppc_path)
```

Comment:
I think specifying the path in this way is inconsistent with the approach we generally take of specifying input paths in `config.toml`. 

Could `ppc_path` (or `oos_path`) perhaps be added as an optional attribute to the MaudConfig object?